lib/as_action_ocaml.ml

```diff
@@ -67,7 +67,7 @@ let compute_deps_ml = compute_deps `Ml
 
 let compile_mli
     ?(needs = []) ?(pkgs = nop) ?(args = []) ~ocamlc ~annot ~incs ~src () =
-  let ctx = As_ctx.v [`OCaml; `Compile; `Src `Mli] in
+  let ctx = As_ctx.v [`OCaml; `Compile; `Src `Mli; `Target `Byte] in
```

Comment:
but it's not clear which ocamlfind command to run then ... byte or native? I guess in practice it doesn't really make a difference...